core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -556,6 +556,9 @@
       "Archive any files/directories instead of moving to the HDFS trash or deleting."),
   GC_TRACE_PERCENT("gc.trace.percent", "0.01", PropertyType.FRACTION,
       "Percent of gc cycles to trace"),
+  GC_USE_FULL_COMPACTION("gc.use.full.compaction", "true", PropertyType.BOOLEAN,
```

Comment:
It might be better to make this more like the durability setting, with three options: `COMPACT`, `FLUSH`, `NONE`
It could be called `gc.post.action` or similar.